app.js

```diff
@@ -0,0 +1,125 @@
+// var breakfast = new Images('breakfast', 'img/breakfast.jpg');
+// var bubblegum = new Images('bubblegum', 'img/bubblegum.jpg');
+// var chair = new Images('chair', 'img/chair.jpg');
+// var cthulhu = new Images('cthulhu', 'img/cthulhu.jpg');
+// var dogduck = new Images('dogduck', 'img/dog-duck.jpg');
+// var dragon = new Images('dragon', 'img/dragon.jpg');
+// var pen = new Images('pen', 'img/pen.jpg');
+// var petsweep = new Images('petsweep', 'img/pet-sweep.jpg');
+// var scissors = new Images('scissors', 'img/scissors.jpg');
+// var shark = new Images('shark', 'img/shark.jpg');
+// var sweep = new Images('sweep', 'img/sweep.png');
+// var tauntaun = new Images('tauntaun', 'img/tauntaun.jpg');
+// var unicorn = new Images('unicorn', 'img/unicorn.jpg');
+// var usb = new Images('usb', 'img/usb.gif');
+// var watercan = new Images('watercan', 'img/water-can.jpg');
+// var wineglass = new Images('wineglass', 'img/wine-glass.jpg');
+
+var imageOne = document.getElementById('image-one');
+var imageTwo = document.getElementById('image-two');
+var imageThree = document.getElementById('image-three');
+
+function randNumGenerator() {
+  while (true) {
+    var randNumOne = Math.floor(Math.random() * 20);
+    var randNumTwo = Math.floor(Math.random() * 20);
+    var randNumThree = Math.floor(Math.random() * 20);
+
+    if (randNumOne !== randNumTwo && randNumThree !== randNumOne && randNumThree !== randNumTwo) {
+      if (randNumOne !== randNumSet[0] && randNumOne !== randNumSet[1] && randNumOne !== randNumSet[2] && randNumTwo !== randNumSet[0] && randNumTwo !== randNumSet[1] && randNumTwo !== randNumSet[2]  && randNumThree !== randNumSet[0] && randNumThree !== randNumSet[1] && randNumThree !== randNumSet[2]) {
+
+        randNumSet[0] = randNumOne;
+        randNumSet[1] = randNumTwo;
+        randNumSet[2] = randNumThree;
+        console.log('ran num set: ', randNumSet);
+
+        shownImages.push(randNumOne, randNumTwo, randNumThree);
+
+        break;
+      }
+    }
+  }
+}
+
+function randomImgOnDom() {
+  imageOne.src = (allImageObjects[randNumSet[0]]).path;
+  imageTwo.src = (allImageObjects[randNumSet[1]]).path;
+  imageThree.src = (allImageObjects[randNumSet[2]]).path;
+}
+
+function clickImage() {
+  imageOne.onclick = function() {
+    var srcAttr = this.getAttribute('src');
+    chosenImages.push(srcAttr.split('.')[0].split('/')[1]);
+    refreshedImages();
+  };
+
+  imageTwo.onclick = function() {
+    var srcAttr = this.getAttribute('src');
+    chosenImages.push(srcAttr.split('.')[0].split('/')[1]);
+    refreshedImages();
+  };
+
+  imageThree.onclick = function() {
+    var srcAttr = this.getAttribute('src');
+    chosenImages.push(srcAttr.split('.')[0].split('/')[1]);
+    refreshedImages();
+  };
+  console.log('chosen images arr:', chosenImages);
+  console.log('chosen images length:',chosenImages.length);
+}
+
+function refreshedImages() {
+
+  if (counter === 25) {
+    alert('done');
```

Comment:
We want to leave the alerts in week one. I know you're still working on this but add some code here that renders a list of all the items to the html with how many times they were clicked on. Also a good idea to remove the images so that I can't be tempted to click on the some more.